ebs-snapshot.sh

```diff
@@ -92,8 +93,9 @@ set -o pipefail
 ## Variable Declartions ##
 
 # Get Instance Details
-instance_id=$(wget -q -O- http://169.254.169.254/latest/meta-data/instance-id)
-region=$(wget -q -O- http://169.254.169.254/latest/meta-data/placement/availability-zone | sed -e 's/\([1-9]\).$/\1/g')
+#Uncomment this to determine region automatically
+#region=$(wget -q -O- http://169.254.169.254/latest/meta-data/placement/availability-zone | sed -e 's/\([1-9]\).$/\1/g')
+region=us-west-2
```

Comment:
In this case, I needed to hard code it so that I could run it from outside an AWS instance (from where http://169.254.169.254/latest/meta-data/placement/availability-zone is not available).   I don't think it's even necessary since the aws cli will read it from the same place(s) as your keys.